### PR TITLE
WL-3851 : Allow the 2 youtube div attributes through Antisamy

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
We're keeping the high Antisamy level and making changes there instead.

File: kernel-impl/src/main/resources/antisamy/high-security-policy.xml [hunk 1]
```diff
@@ -191,6 +191,19 @@
             </regexp-list>
          </attribute>
 
+        <attribute name="data-src" description="One of the set of data-* attributes used to store custom data private to the page or application">
+            <regexp-list>
+                <regexp name="anything" />
```
Comment (buckett) [inline kernel-impl/src/main/resources/antisamy/high-security-policy.xml:196]: Does this mean anything in the attribute value, or anything at all?

Comment (nicholaswilson100) [inline kernel-impl/src/main/resources/antisamy/high-security-policy.xml:196]: 'anything' here refers to another rule (line 35) which says that attribute's value can match anything.

<regexp name="anything" value=".*" />

Some other ones specify a list of values 

```
            <literal value="screen"/>
            <literal value="tty"/>
            <literal value="tv"/>
            <literal value="projection"/>
```

or a regex.

**Comment by buckett** [comment]
All good, nice.

**Pull request merged**